backend/ee/danswer/background/celery/celery_app.py

```diff
@@ -103,5 +111,9 @@ def autogenerate_usage_report_task() -> None:
         "task": "autogenerate_usage_report_task",
         "schedule": timedelta(days=30),  # TODO: change this to config flag
     },
+    "check-ttl-management": {
+        "task": "check_ttl_management_task",
+        "schedule": timedelta(seconds=5),
```

Comment:
I don't think we need to do this every 5 seconds. Maybe once every hour?

Reply:
I agree, yuhong suggested 1 hour but doesnt seem super urgent.